src/charm.py

```diff
@@ -180,9 +191,9 @@ def _render_conf_file(self) -> str:
 
         rendered = template.render(
             db_info=self._get_database_relation_info(),
-            consent_url=join(self.config.get("login_ui_url", ""), "consent"),
-            error_url=join(self.config.get("login_ui_url", ""), "oidc_error"),
-            login_url=join(self.config.get("login_ui_url", ""), "login"),
```

Comment:
the config option `login_ui_url` should be removed from `config.yaml`

Reply:
done